README.md

```diff
@@ -20,22 +20,26 @@ If you like my work, please feel free to provide a personal donation
 [![Donate](https://raw.githubusercontent.com/DrozmotiX/ioBroker.tado/main/admin/button.png)](http://paypal.me/DutchmanNL)
 
 ## Things you can steer
-| State | Description |
-| ----- | ----------- |
-| tado.[x].[yyyyyy].Rooms.[z].setting.power | Turn device on/off |
-| tado.[x].[yyyyyy].Rooms.[z].setting.temperature.celsius | Define temperature |
-| tado.[x].[yyyyyy].Rooms.[z].overlayClearZone | Switch to automatic mode |
-| tado.[x].[yyyyyy].Rooms.[z].overlay.termination.typeSkillBasedApp | Set time table mode |
-| tado.[x].[yyyyyy].Rooms.[z].overlay.termination.durationInSeconds | Set how long the time table mode shall apply |
-| tado.[x].[yyyyyy].Rooms.[z].devices.[RUaaaaaaaaaa].offset.offsetCelsius | Temperature offset |
-| tado.[x].[yyyyyy].Rooms.[z].timeTables.tt_id | Select active time table |
-| tado.[x].[yyyyyy].Home.state.presence | Set HOME or AWAY mode |
-| tado.[x].[yyyyyy].Home.masterswitch | Turn all devices on/off |
-| tado.[x].[yyyyyy].Rooms.[z].setting.mode | AC mode (only AC devices) |
-| tado.[x].[yyyyyy].Rooms.[z].setting.fanspeed | Fanspeed (only AC devices with V3 and older versions) |
-| tado.[x].[yyyyyy].Rooms.[z].setting.fanLebel | Fanlebel (only AC devices with V3+ version) |
-| tado.[x].[yyyyyy].Rooms.[z].setting.verticalSwing | Vertical swing (only AC devices with V3+ version) |
-| tado.[x].[yyyyyy].Rooms.[z].setting.horizontalSwing | Horizontal swing (only AC devices with V3 and olderversions) |
+| State                                                                   | Description                                                                                                  |
+|-------------------------------------------------------------------------|--------------------------------------------------------------------------------------------------------------|
+| tado.[x].[yyyyyy].Rooms.[z].setting.power                               | Turn device on/off                                                                                           |
```

Comment:
Which advantage do you see in adding the whites paces?
On my laptop screen one row is now displayed in 2 lines, which makes it harder to read

Reply:
Wasn't done on purpose, IntelliJ seems to update this automatically. As long as you don't wrap lines in the IDE, it increases readability as all columns are aligned. I changed it back